.cascade-code/Chapman.edu/_cascade/formats/modular/1 Column Primary Content.vtl

```diff
@@ -28,79 +27,83 @@
 ## Widgets
 #foreach ($widget in $widgets)
     #set ($widgetType = $widget.getChild('widgetType').value)
-    
+
     #if ($widgetType == 'Fact Cards with Text')
         #outputFactCardsWithText($widget)
     #end
-    
+
     #if ($widgetType == 'Image with Text')
         #outputImageWithText($widget)
     #end
-    
+
     #if ($widgetType == 'Video with Text')
         #outputVideoWithText($widget)
     #end
-    
+
     #if ($widgetType == 'Text Only')
         #outputTextOnly($widget)
     #end
 
     #if ($widgetType == 'Call To Action 3 Up')
         #outputCallToAction3Up($widget)
     #end
-    
+
     #if ($widgetType == 'Social.chapman Feed')
         #outputChapmanSocialFeed($widget)
     #end
-    
+
     #if ($widgetType == 'Subscribe')
         #outputSubscribe($widget)
     #end
-    
+
     #if ($widgetType == 'Call To Action Block')
         #outputCallToActionBlock($widget)
     #end
-    
+
     #if ($widgetType == 'Events.chapman Feed')
         #outputChapmanEventsFeed($widget)
     #end
-    
+
     #if ($widgetType == 'Stories Feed')
         #outputChapmanStoriesFeed($widget)
     #end
-    
+
     #if ($widgetType == 'Call To Action Footer')
         #outputCallToActionFooter($widget)
     #end
-    
+
     #if ($widgetType == 'Contact Footer')
         #outputContactFooter($widget)
     #end
-    
+
     #if ($widgetType == 'Multi Image Content Blocks')
-        #outputMultiImageContentBlocks($widget)
+        #set ($displayToggle =
```

Comment:
This is interesting. It might be a nice idea to implement this as a standard option across the board if it makes sense